README.md

```diff
@@ -20,6 +20,14 @@ If you want to interrogate germline variants and their relation to cancer predis
 
 ### News
 
+-   *March 2022*: **1.0.1 release**
+
+    -   Fixed bug for huge input sets that cause JSON output crash
+        - huge input variant sets (WGS) are now reduced prior to reporting with R, i.e. exclusion of intronic and intergenic variants, as well as upstream/downstream gene variants
```

Comment:
```suggestion
        - huge input variant sets (WGS) are now reduced prior to reporting with R, i.e. exclusion of intronic and intergenic variants, as well as upstream/downstream gene variants ([#178](https://github.com/sigven/pcgr/issues/178)).
```

Reply:
Actually, I'd suggest keeping just a super short summary about changes in the README News, and instead focus on more detail in the CHANGELOG. So perhaps just a 'Fixed #123 and #456'.

Reply:
Good point, Peter. I'll stick to that habit from now on, but will here accept your current suggestions:-)